go_modules/lib/dependabot/go_modules/update_checker/latest_version_finder.rb

```diff
@@ -94,8 +94,8 @@ def fetch_available_versions
               env = { "GOPRIVATE" => @goprivate }
 
               versions_json = SharedHelpers.run_shell_command(
-                "go list -m -versions -json #{dependency.name}",
-                fingerprint: "go list -m -versions -json <dependency_name>",
+                "go list -e -m -versions -json #{dependency.name}",
+                fingerprint: "go list -e -m -versions -json <dependency_name>",
                 env: env
               )
               version_strings = JSON.parse(versions_json)["Versions"]
```

Comment:
What do you think about adding an `error = JSON.parse(versions_json)["Error"]["Err"]` here and log that as a warning?

Reply:
I wonder if this would be a better longer term fix. If the error comes back in the JSON we know it was a resolvability error and don't need to maintain regex for error messages. If it is a HelperSubprocessFailed error it's probably something on our side we need to fix.